src/rng.asm

```diff
@@ -147,7 +147,7 @@ hook_botwoon_rng:
 
     LDA !ram_botwoon_rng : BEQ .no_manip
 
-    DEC : ASL #3
+    DEC : ASL #3 : INC
```

Comment:
Link to old menu source with needed values for RNG manip: https://github.com/tewtal/sm_practice_hack/blob/12cc9c87c421c8a39720a74ed210c1444f424787/menu.asm#L1066-L1071

`DEC : ASL : ASL : ASL : INC`
8 * (X - 1) + 1 = !ram_botwoon_rng
Random = 0
Down = 1 -> 1
Up = 2 -> 9
Right = 3 -> 17
Left = 4 -> 25